src/templates/pycontw-2018/base.html

```diff
@@ -12,22 +12,23 @@
 <meta name="viewport" content="width=device-width, initial-scale=1">
 
 {% block titletag %}
-<title>{% block title %}{% endblock %} | PyCon Taiwan 2018</title>
+<title>{% block title %}{% endblock title %} | PyCon Taiwan 2018</title>
 {% endblock titletag %}
 
-<link rel="icon" type="image/png" href="{% static 'pycontw-2018/images/favicon.png' %}">
+<link rel="shortcut icon" href="{% static 'pycontw-2018/assets/favicon.ico' %}">
+<link rel="icon" type="image/png" href="{% static 'pycontw-2018/assets/favicon-16x16.png' %}" sizes="16x16">
+<link rel="icon" type="image/png" href="{% static 'pycontw-2018/assets/favicon-32x32.png' %}" sizes="32x32">
 
 {% block meta_og %}
 <meta property="og:description" content="{% block meta_description %}{% endblock meta_description %}">
 <meta property="og:image" content="{{ request.scheme }}://{{ request.get_host }}{% static 'pycontw-2018/images/og-image.png' %}">
```

Comment:
Just noting: image file `pycontw-2018/images/og-image.png` is deleted in this PR.

Reply:
Need to be re-added (Carol provided the source on Slack).